lua/auto-session/init.lua

```diff
@@ -153,6 +151,27 @@ local function bypass_save_by_filetype()
   return true
 end
 
+local function close_ignored_filetypes()
```

Comment:
I have been trying to keep init.lua cleaner by moving utility functions to lib. I think it would be cleaner to move this to Lib and have it take a table. e.g.:

```lua
---Close any buffers that have a ft that is in ignored_filetypes
---@param ignored_filetypes table list of filetypes to close
function Lib.close_ignored_filetypes(ignored_filetypes)
```